lib/widgets/workouts/day.dart

```diff
@@ -137,6 +145,7 @@ class _WorkoutDayWidgetState extends State<WorkoutDayWidget> {
 
   @override
   Widget build(BuildContext context) {
+    _sets.sort((a, b) => a.order!.compareTo(b.order!));
```

Comment:
We would also use the sets in other parts of the application, it would be better to put that in the provider so that the correct order is used there as well. I think it's enough if we sort all the sets every time we add/delete one or call reorderSets

Reply:
I made the changes to addition and deletion sections. `reorderSets` loops and sets the order; I don't think we need to sort it there. Let me know if something is off.